### PR TITLE
feat: continue activities and plan activities

### PR DESCRIPTION
**Pull request by vjnieve**

### CODE REVIEW

File: src/screens/ActivityActions/ContinueActivity/Item.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  };
+
+  const handleStopActivity = (activity) => {
+    const {
+      _id,
+      time: { paused },
+    } = activity;
+
+    console.log(_id);
+    console.log(last(paused).time);
```
Comment (buzzpsych) [inline src/screens/ActivityActions/ContinueActivity/Item.js:34]: console not needed

File: src/screens/ActivityActions/Timer/index.js [hunk 2]
```diff
@@ -95,6 +101,7 @@ const Timer = () => {
   React.useEffect(() => {
     if (stopActivityError) console.log(stopActivityError);
     if (stopActivityData) {
+      console.log("joinin subcription ", stopActivityData);
```
Comment (buzzpsych) [inline src/screens/ActivityActions/Timer/index.js:104]: console not needed

File: src/screens/User/Settings.js [hunk 3]
```diff
@@ -1,11 +1,138 @@
...
+import { GET_USER } from "../../graphql/queries/user/getUser";
+import { api } from "../../common/constants";
 
+const list = ["Activities", "Projects"];
 const Settings = () => {
+  const userSession = useRecoilValue(userState);
+
+  const [updateUser] = useMutation(UPDATE_USER, {
+    onCompleted: () => {
+      toast.show("Profile updated", { type: "success" });
```
Comment (buzzpsych) [inline src/screens/User/Settings.js:19]: Where does toast come from?

Comment (vjnieve) [reply src/screens/User/Settings.js:19]: from project root we just need to declare it there and we can use it everywhere 
image

Comment (buzzpsych) [reply src/screens/User/Settings.js:19]: that's awesome

**Pull request merged**